content/release-10-18-0/platform-10-18-0.md

```diff
@@ -437,6 +445,14 @@ Build version</th>
 <td>UI</td>
 </tr>
 
+<tr>
+<td>Messaging Service</td>
+<td>Fixed an issue where requests from the core platform into the Messaging Service could take a long time to complete, slowing down the response to HTTP requests and potentially preventing the platform from handling new incoming requests. For example, a request from the core platform to publish a message using Notifications 2.0 could block if the tenant had reached its quota for unconsumed notifications, only timing out after a long delay. This issue has been resolved by ensuring that Messaging Service requests that would have blocked now time out quickly.</td>

```

Comment:
```suggestion
<td>Fixed an issue where requests from the core platform into the Messaging Service could take a long time to complete, slowing down the response of HTTP requests and potentially preventing the platform from handling new incoming requests. For example, a request from the core platform to publish a message using Notifications 2.0 could block if the tenant had reached its quota for unconsumed notifications, only timing out after a long delay. This issue has been resolved by ensuring that Messaging Service requests that would have blocked now time out quickly.</td>
```

i guess both work, i prefer the suggested

Reply:
Since we already have this entry in various places I will leave as is for consistency.